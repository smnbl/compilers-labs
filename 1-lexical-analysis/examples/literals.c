1 123
1.2 12.
"foo" "bar"
