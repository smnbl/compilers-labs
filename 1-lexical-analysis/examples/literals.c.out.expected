1:1 -> 1:2          1                   INT_LITERAL         
1:3 -> 1:6          123                 INT_LITERAL         
2:1 -> 2:4          1.2                 FLOAT_LITERAL       
2:5 -> 2:8          12.                 FLOAT_LITERAL       
3:1 -> 3:6          "foo"               STRING_LITERAL      
3:7 -> 3:12         "bar"               STRING_LITERAL      
